modules/openvpn/vpn-instance.tf

```diff
@@ -0,0 +1,89 @@
+resource "aws_autoscaling_group" "vpn-asg" {
```

Comment:
VPNs typically need a static IP. With an autoscaling group we might get a new IP and our DNS record might not get updated